sr_grasp_stability/src/superseded/tf_computation.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import rospy
+import tf
+
+
+class TfComputator:
+    """
+    Hand tf transformation computation
+    """
+
+    def __init__(self):
+
+        # Dictionaries to store transformations
+        self.trans = {}
+        self. rot = {}
+
+        self.tf_listener = tf.TransformListener()
+
+    def get_finger_tips(self):
+        """
+        Get fingertip transformations
+
+        () -> return (dict, dict)
+        """
+        # finger-tip frames:
+        # '/rh_rftip', '/rh_lftip', '/rh_mftip', '/rh_thtip', '/rh_fftip',
+
+        exc = False
+        # Get transforms from forearm to distal #
+        try:
+            (self.trans['rh_ffdistal'], self.rot['rh_ffdistal']) = self.tf_listener.lookupTransform(
```

Comment:
I would like the fingertip names to be configurable.

The reason is that we need all this to be compatible with hand H as well.